src/js/tabs/motors.js

```diff
@@ -558,8 +558,12 @@ TABS.motors.initialize = function (callback) {
 
             $('div.values li').eq(index).text($(this).val());
 
-            for (var i = 0; i < 8; i++) {
+            for (var i = 0; i < FC.MOTOR_CONFIG.motor_count; i++) {
                 var val = parseInt($('div.sliders input').eq(i).val());
+                // adjust val to retain the same while switching tabs
+                if (val !== 1000){
```

Comment:
:space_invader: missing space

Reply:
Thanks, will fix after finding root cause.

 Strange thing also pushing the slider up on 1002 it waits for a second click up or down?